commit 9a948c1af9c07bfdc0eb079cc3cbc84e6bd597c6
Author: Timothy Pearson <kb9vqf@pearsoncomputing.net>
Date:   1337062714 -0500

    Disable keyboard shortcuts for file location moving, as they did not work properly and have very little practical use

diff --git a/kate/app/katefilelist.cpp b/kate/app/katefilelist.cpp
index bd2d087..d2214a4 100644
--- a/kate/app/katefilelist.cpp
+++ b/kate/app/katefilelist.cpp
@@ -139,9 +139,9 @@ void KateFileList::setupActions ()
   sortAction = new KSelectAction( i18n("Sort &By"), 0,
       m_main->actionCollection(), "filelist_sortby"  );
   listMoveFileUp = new KAction( i18n("Move File Up"), 0, m_main->actionCollection(), "filelist_move_up" );
-  listMoveFileUp->setShortcut(KShortcut(CTRL + SHIFT + Key_Comma));
+  //listMoveFileUp->setShortcut(KShortcut(CTRL + SHIFT + Key_Comma));
   listMoveFileDown = new KAction( i18n("Move File Down"), 0, m_main->actionCollection(), "filelist_move_down" );
-  listMoveFileDown->setShortcut(KShortcut(CTRL + SHIFT + Key_Period));
+  //listMoveFileDown->setShortcut(KShortcut(CTRL + SHIFT + Key_Period));
   connect( listMoveFileUp, TQT_SIGNAL(activated()), TQT_TQOBJECT(this), TQT_SLOT(moveFileUp()) );
   connect( listMoveFileDown, TQT_SIGNAL(activated()), TQT_TQOBJECT(this), TQT_SLOT(moveFileDown()) );
   TQStringList l;
